scripts/repo-utils/index.cjs

```diff
@@ -15,7 +15,11 @@ try {
   USE_ESM = type === "module";
 } catch (_) {}
 
+function bool(value) {
+  return value && value !== "false" && value !== "0";
+}
 exports.USE_ESM = USE_ESM;
+exports.IS_BABEL_8 = () => bool(process.env.BABEL_8_BREAKING);
```

Comment:
Why is this a function and not just a boolean?

Reply:
We may sometimes set it dynamically.